src/antares/service/local_services/area_local.py

```diff
@@ -334,3 +335,33 @@ def read_areas(self) -> List[Area]:
                     )
                 )
         return areas
+
+    def _read_timeseries(self, ts_file_type: TimeSeriesFileType, study_path: Path, area_id: Optional[str] = None, group_id: Optional[str] = None,constraint_id: Optional[str] = None, time_series: Optional[pd.DataFrame] = None,) -> pd.DataFrame:
+        def _write_file(_file_path: Path, _time_series: pd.DataFrame) -> None:
+            _file_path.parent.mkdir(parents=True, exist_ok=True)
+            _time_series.to_csv(_file_path, sep="\t", header=False, index=False, encoding="utf-8")
+        
+        file_path = study_path / (
+            ts_file_type.value
+            if not (area_id or constraint_id or group_id)
```

Comment:
i don't see a group_id inside the class TimeSeriesFileType, i think you should remove it. And you shouldn't expect a time_series inside a read method

Reply:
Next commit